src/app.rb

```diff
@@ -0,0 +1,98 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './modules/list'
+require_relative './modules/check'
+
+class App
+  def initialize
+    @list_people = []
+    @list_books = []
+    @rentals = []
+  end
+
+  include ShowList
+  include CheckConditions
+
+  def create_person
+    puts 'Do you want to create a Student (1) or a Teacher (2) [Input the number]:'
+    create_person_input = gets.chomp.to_i
+    case create_person_input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Invalid please choose [1] or [2]'
+      create_person
+    end
+  end
+
+  def create_student
+    puts 'Age:'
+    student_age = gets.chomp.to_i
+    age_stud_checker(student_age)
+    student_age = gets.chomp.to_i
+
+    puts 'Name:'
+    student_name = gets.chomp.capitalize
+    label = 'Student'
+
+    puts 'Do you have parent permission? [y/n]:'
+    student_permission = gets.chomp
+    permission_checker(student_permission)
+
+    student = Student.new(student_age, label, student_name, student_permission)
+    @list_people.push(student)
+    puts 'Person created successfully!'
+  end
+
+  def create_teacher
+    puts 'Age:'
+    teacher_age = gets.chomp.to_i
+    age_checker(teacher_age)
+    teacher_age = gets.chomp.to_i
+
+    puts 'Name:'
+    teacher_name = gets.chomp.capitalize
+
+    puts 'Specialization:'
+    teacher_specialization = gets.chomp.capitalize
+
+    teacher = Teacher.new(teacher_age, teacher_name, teacher_specialization)
+    @list_people.push(teacher)
+    puts 'Person created succesfully!'
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix the `error` shown in the below `image` because I am unable to do the **"Create person"** part of the `teacher specialization` part 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/210277390-b6894864-3d23-4500-a64a-f22e273e261c.png)

Reply:
Fixed this too Thanks agaim